### ISSUE TITLE
Installing the package failed

### ISSUE THREAD

**Issue by Mihiretukebede** [doehm/eyedroppeR#2]
`devtools::install_github("doehm/eyedroppeR")
`
Warning message:
In i.p(...) :
  installation of package ‘..../AppData/Local/Temp/RtmpOuo7pj/file30b06c2ca1c/eyedroppeR_0.1.0.tar.gz’ had non-zero exit status

I also tried to download the zip files and install them manually. It didin't work.

**Comment by doehm** [comment]
Could you please try running `remotes::install_github("doehm/eyedroppeR", dependencies = TRUE)`?

**Comment by Mihiretukebede** [comment]
Similar error.

**Comment by doehm** [comment]
Right. I can't recreate the error so it'll be difficult to debug. Could you please install the latest version of R and {rlang}?

**Comment by doehm** [comment]
There may have been an issue with {utils} not being in the imports. Could you please try `remotes::install_github("doehm/eyedroppeR", dependencies = TRUE)` again?

**Comment by Mihiretukebede** [comment]
No it doesn't work
